tests/unit/js/test.subjects.js

```diff
@@ -0,0 +1,80 @@
+const sinon = require('sinon');
+const jquery = require('jquery');
+const { urlencode, Subject,
+    renderTag, slice } = require('../../../openlibrary/plugins/openlibrary/js/subjects');
+let sandbox, spy;
+
+beforeEach(() => {
+    sandbox = sinon.createSandbox();
+    global.$ = jquery;
+    spy = sinon.spy();
+    // Log is added inside utils.js, but we stub it here.
+    $.log = spy;
+    sandbox.stub(global, '$').callsFake(jquery);
+});
+
+describe('subjects: urlencode', () => {
```

Comment:
No need to prefix everything with `subjects` (violates DRY :) )

Reply:
DONE